### ISSUE TITLE
Game Crashes upon Launch (probably incompatible mods)

### ISSUE THREAD

**Issue by dawottyt** [Cosmetica-cc/Cosmetica#143]
Not sure if this is the correct place to put this but I couldn't think of a better one so lmk if theres somewhere else i should post this.

**Describe the bug**
Game crashes upon launch

**To Reproduce**
1. Install Cosmetica and other mods
2. Run Game
3. Game closes on startup

**Expected behavior**
Game Launches as Normal

**Screenshots**

**Setup (please complete the following information):**
 - OS: Windows 10
 - Minecraft Version: 1.20.1
 - Version: All that I've tested

**Additional context**
https://pastebin.com/raw/YsxycFvE

**Comment by valoeghese** [comment]
Yeah the mod that appears in the stack trace is immediatelyfast so I suspect it's a conflict with them
